snapcraft/plugins/go.py

```diff
@@ -39,6 +39,10 @@
       This entry tells the checked out `source` to live within a certain path
       within `GOPATH`.
       This is not needed and does not affect `go-packages`.
+
+    - go-buildtags:
+      (string)
```

Comment:
Agreed. That would make it consistent with the other plugins with similar options, as well.